datasets/flwr_datasets/federated_dataset.py

```diff
@@ -82,6 +94,9 @@ def __init__(
         subset: Optional[str] = None,
         resplitter: Optional[Union[Resplitter, Dict[str, Tuple[str, ...]]]] = None,
         partitioners: Dict[str, Union[Partitioner, int]],
+        partition_division: Optional[
+            Union[List[float], Tuple[float, ...], Dict[str, float]]
+        ] = None,
```

Comment:
Shouldn't this be the last argument b/c conceptually, it's the last thing that happens?

Reply:
My reasoning behind this order is more based on functionality-based grouping + assumed frequency of use. 
Groups: To me, the partition_division is logically connected to the partitioners (the division operation happens after the partitioning), and then the next group is shuffling + seed. 
Frequency: I also expect that the shuffling + seed would require less frequency change of the default parameters than the specification of `partition_division`. That's why I left them last.
That's my reasoning. Wdyt?